packages/application-generic/src/services/queues/index.ts

```diff
@@ -15,4 +15,5 @@ export {
   StandardQueueService,
   WebSocketsQueueService,
   WorkflowQueueService,
+  InboundParseQueueService,
```

Comment:
Better name it in the import than export it twice with different names.

Reply:
What do you mean?

Reply:
With your addition in `[packages/application-generic/src/services/queues/index.ts](https://github.com/novuhq/novu/pull/4180/files/195c10a8a44705b827d0b6d130cad297d6c54b2e#diff-47bdb9862f6d33518064bef8765388edcd2d7182861a8f09a0edc6dba3b3edbd)` we will be exporting twice the same class, as `InboundParseQueueService` and as `InboundParseQueue` (this was done like this because some name clashing in the Inbound Parse Mail app.

So that's why I said that in `packages/application-generic/src/custom-providers/index.ts` when importing in line 13 you should do:
```
InboundParseQueue as InboundParseQueueService
```.

Another options would be to export the original name (what you tried) and rewrite the import in Inbound Parse Mail app. Any direction but export twice the same class with 2 different names.